kaiax/gasless/impl/init.go

```diff
@@ -57,6 +61,20 @@ func (g *GaslessModule) Init(opts *InitOpts) error {
 	g.InitOpts = *opts
 	g.signer = types.LatestSignerForChainID(g.ChainConfig.ChainID)
 
+	// Disable module if node does not have sufficient balance
+	if g.MinBalance != nil {
```

Comment:
How about checking this in `PostInsertBlock` to dynamically turn on/off the GA based on the CN's balance?

Reply:
Dynamic enable/disable is not currently supported. If disabled at `Init()`, the module is simply not registered: https://github.com/kaiachain/kaia/blob/bf9a03390737c8dc778061177aee523e58d8f370/node/cn/backend.go#L563-L569

Reply:
Oh, you're right. Thanks.